primefaces/src/main/resources/META-INF/resources/primefaces/clock/clock.js

```diff
@@ -610,7 +609,7 @@ PrimeFaces.widget.Clock = PrimeFaces.widget.BaseWidget.extend({
     draw_hour_signs: function() {
         this.hour_sign = [];
 
-        for (i = 0; i < 12; i++) {
+        for (let i = 0; i < 12; i++) {
```

Comment:
Looks good. Without let it creates a global variable window.i, which I don't anybody intended

![image](https://github.com/user-attachments/assets/29946cef-3d85-4ebe-bd1d-062ecd6f41b5)